app/api/events.py

```diff
@@ -104,6 +105,14 @@ class Meta:
                                  schema='EventInvoiceSchema',
                                  many=True,
                                  type_='event_invoice')
+    discount_codes = Relationship(attribute='discount_code',
```

Comment:
@enigmaeth Event can have multiple discount codes. I guess discount_code_id field in event model should be removed then